index.js

```diff
@@ -1,8 +1,8 @@
 module.exports = addCorsHeaders
 
-function addCorsHeaders (request, reply) {
+function addCorsHeaders (request, h) {
   if (!request.headers.origin) {
-    return reply.continue()
+    return h.continue
```

Comment:
could you check here if `h.continue` is a function and if it is, `return h.continue()`, otherwise `return h.continue`?

Reply:
they are both functions. 
The Hapi team, made the call different. The plugins that works on 17+ wont work on earlier versions. So most of the plugins have a break change version. Like 1.0 works with 16- 2.0 17+

Reply:
Okay, got it. It’s fine, I can release a breaking version. Can you check that the tests pass? Please compare with the other open PRs 🙏 Thanks!